scripts/rabbitmq-server-ha.ocf

```diff
@@ -1537,7 +1571,45 @@ get_monitor() {
         return $rc
     fi
 
-    # Check if the rabbitmqctl control plane is alive.
+    node_health_check
+    rc=$?
+
+    return $rc
+}
+
+# Check if the rabbitmqctl control plane is alive.
+node_health_check() {
+    if [ "$OCF_RESKEY_rmq_feature_health_check" = true ]; then
+        node_health_check_local
+    else
+        node_health_check_legacy
+    fi
+}
+
+node_health_check_local() {
+    local LH="${LH} node_health_check_local():"
+    local rc
+    local rc_timeouts
+
+    local timeout=$((TIMEOUT_ARG - 2)) # give node_health_check some time to handle timeout by itself
+    su_rabbit_cmd "${OCF_RESKEY_ctl} node_health_check -t $timeout"
```

Comment:
Is there a reason to use timeout built-into rabbitmqctl? I think we can use only /usr/bin/timeout as we do right now just for simplicity.